skyportal/utils/notifications.py

```diff
@@ -65,24 +69,27 @@ def gcn_notification_content(target, session):
         source_name = f"GW{dateobs_txt[2:4]}{dateobs_txt[5:7]}{dateobs_txt[8:10]}.{dateobs_txt[11:13]}{dateobs_txt[14:16]}{dateobs_txt[17:19]}"
 
     if notice_content is not None:
-        loc = notice_content.find('./WhereWhen/ObsDataLocation/ObservationLocation')
-        ra = loc.find('./AstroCoords/Position2D/Value2/C1')
-        dec = loc.find('./AstroCoords/Position2D/Value2/C2')
-        error = loc.find('./AstroCoords/Position2D/Error2Radius')
+        if isinstance(notice_content, dict):
+            ra = notice_content.get("ra")
+            dec = notice_content.get("dec")
+            error = notice_content.get("ra_dec_error")
+        else:
+            loc = notice_content.find('./WhereWhen/ObsDataLocation/ObservationLocation')
+            ra = loc.find('./AstroCoords/Position2D/Value2/C1')
+            dec = loc.find('./AstroCoords/Position2D/Value2/C2')
+            error = loc.find('./AstroCoords/Position2D/Error2Radius')
+
+            if ra is not None:
```

Comment:
same comment here. Any reason not to apply this dtype conversion to both cases?

Reply:
same deal here.